### PR TITLE
[add][s] Adding a github workflow

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:

- Added Github Automation workflow

cc @anuveyatsu

### CODE REVIEW

**Review by anuveyatsu — APPROVED** [review]
LGTM 👍🏼  happy to merge once #8 is merged

**Comment by gradedSystem** [comment]
@anuveyatsu this could be run it will automatically update the data in the `data/annual.csv` and `data/monthly.csv` files

**Pull request merged**
